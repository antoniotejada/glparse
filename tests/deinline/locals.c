//
// Check that functions are deinlined even with local declarations and empty lines
// in between
//

void a()
{

}

void f()
{
    GLchar local_char_ptr_0[] = "position";
    a();
    float local_float_ptr_1[] = { 0.00249999994412, 0.0, 0.0, 0.0, 0.0, -0.00165975105483, 0.0, 0.0, 0.0, 0.0, -1.0, 0.0, -1.0, 1.0, -0.0, 1.0 };
    a();
    const char local_const_char_ptr_2[64] = { 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x80, 0x3f };
    a();

    a();

    a();

    a();
}