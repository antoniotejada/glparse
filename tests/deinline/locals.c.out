// Src lines 9
// Dest lines 8
//
// Check that functions are deinlined even with local declarations and empty lines
// in between
//

void a();
void f();
void subframe2();

void a()
{
}

void f()
{
    GLchar local_char_ptr_0[] = "position";
    float local_float_ptr_1[] = { 0.00249999994412, 0.0, 0.0, 0.0, 0.0, -0.00165975105483, 0.0, 0.0, 0.0, 0.0, -1.0, 0.0, -1.0, 1.0, -0.0, 1.0 };
    const char local_const_char_ptr_2[64] = { 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x80, 0x3f, 0x0, 0x0, 0x80, 0x3f };
    subframe2();
    subframe2();
    subframe2();
}

void subframe2()
{
    a();
    a();
}
